### DISCUSSION TITLE
How to suppress an issue for a particular line

### DISCUSSION THREAD

**Discussion by dennisdoomen** [JetBrains/Qodana#199] [Q&A]
We have a couple of examples where we don't agree with the `invertif` rule. I know we can completely disable the rule, but how can we suppress the individual problems for the specific lines? See https://github.com/fluentassertions/fluentassertions/pull/2359/files#diff-25f46ce96c5793b6343ba816b35ad07012d1eb6113f6ff4e0f1a250b4d27f177R49.

Qodana Cloud doesn't offer this option either

![image](https://github.com/JetBrains/Qodana/assets/572734/abb1f1d8-114f-46c3-ad37-a21463a92bfd)

**Comment by brichbash** [comment]
Hello,
You can do that by adding the following comment to the line you want to suppress:
```
// ReSharper disable once invertif 
if (success)
...
```

**Reply by dennisdoomen** [reply]
Hmm, is there a way to do this through the qodana config?

**Reply by brichbash** [reply]
Unfortunately, there is not. We reuse the inspection engines of our IDEs, and most of them work the same way: to suppress an inspection for a class/method/line add a comment/annotation with an inspection id.
